src/main/groovy/io/gatling/gradle/GatlingEnterpriseStartTask.groovy

```diff
@@ -9,46 +12,79 @@ import org.gradle.api.tasks.TaskExecutionException
 
 @CacheableTask
 class GatlingEnterpriseStartTask extends DefaultTask {
-
     @Internal
     Closure simulations
 
     @TaskAction
     void publish() {
         def gatling = project.extensions.getByType(GatlingPluginExtension)
-        EnterpriseClient enterpriseClient = gatling.enterprise.getEnterpriseClient(project.version.toString())
 
         def systemProps = gatling.enterprise.systemProps ?: [:]
 
+        def simulationAndRunSummary = gatling.enterprise.simulationId
+            ? startNonInteractive(gatling, systemProps)
+            : gatling.enterprise.batchMode
+                ? createNonInteractive(gatling, systemProps)
+                : createOrStartInteractive(gatling, systemProps)
```

Comment:
Handle `SimulationStartException` may need the two creation methods to be regroup, so that they can be wrap in a try catch logging that simulation have been created, then rethrow the cause. (see [SBT implementation](https://github.com/gatling/gatling-sbt-plugin/blob/917dfe72a04d531f96520a11757f3171bab80a65/src/main/scala/io/gatling/sbt/settings/gatling/EnterpriseSettings.scala#L295))

```suggestion
        def simulationAndRunSummary = gatling.enterprise.simulationId
            ? startNonInteractive(gatling, systemProps)
            : createOrStart(gatling, systemProps)
```